pages.pt_BR/common/[.md

```diff
@@ -0,0 +1,25 @@
+# [
+
+> Avalia condição.
+> Retorna 0 se a condição for verdadeira, 1 se for falsa.
```

Comment:
"for" in Portuguese?  Isn't it something like `para`? (maybe that's Spanish though 🤔)

Reply:
Portuguese is grammatically complex :laughing:

"For" is a flexion of the verb "ser" (be)

A simple translation for: "If x **is** y"
Would be: "Se x **é** y"

However, since it is and condition, the correct conjugation is: "Se x **for** y"

That is all the possible conjugations for the verb "ser" (be) in pt-BR:

![image](https://user-images.githubusercontent.com/60361387/116953413-90710d80-ac63-11eb-80ad-042544fdd398.png)

So, yeah, I think it is correct.

Reply:
Oh, I see!  That makes sense.  I thought that you accidentally forgot to translate "for," but now I see that "for" isn't even in the English page 🤣

Reply:
I appreciate your careful review!